composer.json

```diff
@@ -21,21 +21,26 @@
         }
     ],
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^7.1",
         "doctrine/phpcr-bundle": "^1.3 || ^2.0",
-        "doctrine/phpcr-odm": "^1.4 || ^2.0",
+        "doctrine/phpcr-odm": "^1.4.2 || ^2.0",
         "sonata-project/admin-bundle": "^3.30.1",
         "sonata-project/block-bundle": "^3.11",
         "symfony-cmf/resource-rest-bundle": "^1.0.1",
         "symfony-cmf/tree-browser-bundle": "^2.0",
-        "symfony/config": "^2.8 || ^3.2",
-        "symfony/dependency-injection": "^2.8 || ^3.2",
-        "symfony/form": "^2.8 || ^3.2",
-        "symfony/framework-bundle": "^2.8 || ^3.2",
-        "symfony/http-foundation": "^2.8 || ^3.2",
-        "symfony/http-kernel": "^2.8 || ^3.2",
-        "symfony/options-resolver": "^2.8 || ^3.2",
-        "symfony/property-access": "^2.8 || ^3.2"
+        "symfony/config": "^2.8 || ^3.2 || ^4.0",
+        "symfony/dependency-injection": "^2.8 || ^3.2 || ^4.0",
+        "symfony/finder": "^2.8 || ^3.2 || ^4.0",
+        "symfony/form": "^2.8 || ^3.2 || ^4.0",
+        "symfony/framework-bundle": "^2.8 || ^3.2 || ^4.0",
+        "symfony/http-foundation": "^2.8 || ^3.2 || ^4.0",
+        "symfony/http-kernel": "^2.8 || ^3.2 || ^4.0",
+        "symfony/options-resolver": "^2.8 || ^3.2 || ^4.0",
+        "symfony/property-access": "^2.8 || ^3.2 || ^4.0"
+    },
+    "conflict": {
+        "sebastian/environment": "<1.3.4",
+        "sebastian/exporter": "<2.0.0"
```

Comment:
these are not real conflicts i guess, but constraints to make the lowest verison tests not fail. if so, they should go into require-dev as this would prevent installing this bundle in a project with a lower version of those sebastian packages.

Reply:
We should move them to `require-dev` 👍